src/main.ts

```diff
@@ -13,10 +13,6 @@ import store from './store';
 
 initializeReporter();
 
-process.on('unhandledRejection', (error) => {
```

Comment:
Why?
In this way we logging async exceptions

Reply:
I thought you asked to remove it in the other pr(#66). I think that if you don't have an unhandledRejection handler it will log the error anyway, but I will revert this change. So what did you mean in this comment https://github.com/brafdlog/budget-tracking/pull/66#discussion_r460475651?

Reply:
I just notice that `lint` mark it as an error. I'm seeing this in the Github interface:

![image](https://user-images.githubusercontent.com/17686879/88938145-b2310480-d28d-11ea-8bfb-4ef98ea7e853.png)

You just need to add ESLint ignore

Reply:
ah, I didn't see this error because it is set to be ignored in dev environment.

I am changing this to be a warning in non prod environment so it will be more visible.